lodash does not get loaded when installing via npm install

When running the example I get the following error:

```
npm run example

> opentracing@0.14.1 example /Users/cwat-areitbau/workspace/attic/node_modules/opentracing
> node lib/examples/demo/demo.js

module.js:471
    throw err;
    ^

Error: Cannot find module 'lodash'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/cwat-areitbau/workspace/attic/node_modules/opentracing/lib/mock_tracer/mock_span.js:14:9)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)

npm ERR! Darwin 17.4.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "example"
npm ERR! node v6.9.5
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! opentracing@0.14.1 example: `node lib/examples/demo/demo.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the opentracing@0.14.1 example script 'node lib/examples/demo/demo.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the opentracing package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node lib/examples/demo/demo.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs opentracing
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls opentracing
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/cwat-areitbau/workspace/attic/node_modules/opentracing/npm-debug.log

```

--- a/package.json
+++ b/package.json
@@ -33,7 +33,6 @@
   "dependencies": {},
   "devDependencies": {
     "@types/chai": "3.4.35",
-    "@types/lodash": "4.14.61",
     "@types/mocha": "2.2.40",
     "@types/node": "7.0.12",
     "chai": "^3.4.1",
@@ -43,7 +42,6 @@
     "cross-env": "^4.0.0",
     "istanbul": "^0.4.5",
     "json-loader": "^0.5.4",
-    "lodash": "^4.17.4",
     "mocha": "^2.4.5",
     "shelljs": "^0.5.3",
     "source-map-support": "^0.3.3",

--- a/src/mock_tracer/mock_report.ts
+++ b/src/mock_tracer/mock_report.ts
@@ -1,5 +1,4 @@
 /* eslint-disable import/no-extraneous-dependencies */
-import * as _ from 'lodash';
 import { DebugInfo, MockSpan } from './mock_span';
 
 /**
@@ -22,7 +21,7 @@ export class MockReport {
 
         this.unfinishedSpans = [];
 
-        _.each(spans, span => {
+        spans.forEach(span => {
             if (span._finishMs === 0) {
                 this.unfinishedSpans.push(span);
             }
@@ -30,7 +29,10 @@ export class MockReport {
             this.spansByUUID[span.uuid()] = span;
             this.debugSpans.push(span.debug());
 
-            _.each(span.tags(), (val, key: string) => {
+            const tags = span.tags();
+
+            Object.keys(tags).forEach((key: string) => {
+                const val = tags[key];
                 this.spansByTag[key] = this.spansByTag[key] || {};
                 this.spansByTag[key][val] = this.spansByTag[key][val] || [];
                 this.spansByTag[key][val].push(span);

--- a/src/mock_tracer/mock_span.ts
+++ b/src/mock_tracer/mock_span.ts
@@ -1,6 +1,5 @@
 /* eslint-disable import/no-extraneous-dependencies */
 
-import * as _ from 'lodash';
 import * as opentracing from '../index';
 import Reference from '../reference';
 import MockContext from './mock_context';
@@ -115,7 +114,7 @@ export class MockSpan extends opentracing.Span {
             operation : this._operationName,
             millis    : [this._finishMs - this._startMs, this._startMs, this._finishMs]
         };
-        if (_.size(this._tags)) {
+        if (Object.keys(this._tags).length) {
             obj.tags = this._tags;
         }
         return obj;

